tests/test_lca.py

```diff
@@ -1153,6 +1153,56 @@ def test_index_and_classify_internal_unassigned_multi():
         assert 'loaded 1 LCA databases' in err
 
 
+@utils.in_tempdir
+def test_index_and_classify_internal_unassigned_multi_2(c):
```

Comment:
pls put `majority` in the test name, and add a comment as the first line of the function that says something like `test --majority-vote option`